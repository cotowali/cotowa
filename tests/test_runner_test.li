namespace mod {
  #[test]
  fn test_ok() {
    assert(true)
  }

  #[test]
  fn test_fail() {
    assert(false)
  }
}
