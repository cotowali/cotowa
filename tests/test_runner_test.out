testing mod::test_fail ... FAILED

Output:
LINE 4: Assertion Failed

testing mod::test_ok ... OK
testing test_fail ... FAILED

Output:
LINE 15: Assertion Failed

testing test_ok ... OK
test result: FAILED
4 total; 2 passed; 2 failed;
